### ISSUE TITLE
Bug - Failing to fix missing quote of a value in an object inside an array

### ISSUE THREAD

**Issue by kazazor** [josdejong/jsonrepair#131]
```js
{
  "timestampRangesMinSec": [
    {"startMinSec": "00:00", "endMinSec": "00:10"},
    {"startMinSec": "00:11", "endMinSec": "00:16"},
    {"startMinSec": "00:17", "endMinSec": "00:23}
  ]
}
```

Notice thet `00:23` is missing closing quotes and the tools breaks with:
> Object key expected at position 166

Test it out here:
https://josdejong.github.io/jsonrepair/

**Comment by josdejong** [comment]
Thanks for reporting, it would be nice to be able to repair this too. This issue is the same as #130: strings containing a delimiter like a colon `:` cannot be repaired. I'll close this issue in favor of #130.

**Comment by josdejong** [comment]
This issue has been fixed in `jsonrepair@3.9.0`.
